### PR TITLE
Update spideroak.rb

### PR DESCRIPTION
**Pull request by soredake**
SpiderOakONE is out

### CODE REVIEW

File: Casks/spideroak.rb [hunk 1]
```diff
@@ -7,5 +7,7 @@
   homepage 'https://spideroak.com/'
   license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
 
-  app 'SpiderOak.app'
+  pkg 'SpiderOakONE.pkg'
+
+  uninstall :delete => '/Applications/SpiderOakONE.app'
```
Comment (vitorgalvao) [inline Casks/spideroak.rb]: I doubt the installer only does this. There are [scripts](https://github.com/caskroom/homebrew-cask/tree/master/developer/bin) and [documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#uninstall-stanza-details) geared to find what is needed. Do you need any help?

Comment (fenque) [inline Casks/spideroak.rb]: I checked, the installer simply copies the program in /Applications, pkgutil --pkgs show anything related to spideroak.

**Comment by vitorgalvao** [comment]
Name has to be changed to `spideroakone`. The [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_token_reference.md) must be followed, to avoid duplicates.

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)? @mention me after so I know to check back.

**Comment by fenque** [comment]
@vitorgalvao done.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as c317dcbe262ef4bd6ddd039ae1512f8c091adee9 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**